doc: sync doc about enable auth for docker image (#324)
apache/incubator-hugegraph-doc: content/cn/docs/quickstart/hugegraph-loader.md

---
title: "HugeGraph-Loader Quick Start"

> linkTitle: "使用 HugeGraph-Loader 导入数据"

Reviewer: ```suggestion
linkTitle: "使用 Loader/Spark 实时导入数据"
```

> 
> 1. hugegraph-loader 的 docker 镜像是一个便捷版本，用于快速启动 loader，并不是**官方发布物料包方式**。你可以从 [ASF Release Distribution Policy](https://infra.apache.org/release-distribution.html#dockerhub) 中得到更多细节。
> 
> 2. 推荐使用 `release tag`(如 `1.2.0`) 以获取稳定版。使用 `latest` tag 可以使用开发中的最新功能。

#### 2.2 下载已编译的压缩包

下载最新版本的 HugeGraph-Toolchain Release 包，里面包含了 loader + tool + hubble 全套工具，如果你已经下载，可跳过重复步骤

```bash
wget https://downloads.apache.org/incubator/hugegraph/{version}//apache-hugegraph-toolchain-incubating-{version}.tar.gz
tar zxf *hugegraph*.tar.gz
```

#### 2.3 克隆源码编译安装

克隆最新版本的 HugeGraph-Loader 源码包：

```bash
# 1. get from github
git clone https://github.com/apache/hugegraph-toolchain.git

> # 2. get from direct  (e.g. here is 1.0.0, please choose the latest version)

Reviewer: ```suggestion
# 2. get from direct url (please choose the **latest release** version)
```

wget https://downloads.apache.org/incubator/hugegraph/{version}/apache-hugegraph-toolchain-incubating-{version}-src.tar.gz
```

由于 Oracle ojdbc license 的限制，需要手动安装 ojdbc 到本地 maven 仓库。
访问[Oracle jdbc 下载](https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html) 页面。选择 Oracle Database 12c Release 2 (12.2.0.1) drivers，如下图所示。

打开链接后，选择“ojdbc8.jar”

把 ojdbc8 安装到本地 maven 仓库，进入`ojdbc8.jar`所在目录，执行以下命令。
```
mvn install:install-file -Dfile=./ojdbc8.jar -DgroupId=com.oracle -DartifactId=ojdbc8 -Dversion=12.2.0.1 -Dpackaging=jar

> ```

Reviewer: 考虑折叠 ojdbc 的内容, 用户已经很少了 (后续不影响编译的话可以进一步压缩)

> 
> 1. hugegraph-loader 的 docker 镜像是一个便捷版本，用于快速启动 loader，并不是**官方发布物料包方式**。你可以从 [ASF Release Distribution Policy](https://infra.apache.org/release-distribution.html#dockerhub) 中得到更多细节。
> 
> 2. 推荐使用 `release tag`(如 `1.2.0`) 以获取稳定版。使用 `latest` tag 可以使用开发中的最新功能。

#### 2.2 下载已编译的压缩包

> 下载最新版本的 HugeGraph-Toolchain Release 包，里面包含了 loader + tool + hubble 全套工具，如果你已经下载，可跳过重复步骤

Reviewer: ```suggestion
下载最新版本的 `HugeGraph-Toolchain` Release 包，里面包含了 `loader + tool + hubble` 全套工具，如果你已经下载，可跳过重复步骤
```
